<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditVideoForm</class>
 <widget class="QWidget" name="EditVideoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VibeTube - Edit Video</string>
  </property>

  <!-- SAME STYLE AS UPLOAD VIDEO -->
  <property name="styleSheet">
   <string notr="true">
    QWidget {
        background-color: #FFE4E9;
        font-family: "Segoe UI";
        font-size: 12pt;
        color: #333333;
    }
    QPushButton {
        background-color: #FF6F91;
        color: white;
        font-weight: bold;
        border-radius: 8px;
        padding: 6px 12px;
    }
    QPushButton:hover {
        background-color: #FF9AA2;
    }
    QLineEdit, QTextEdit, QComboBox {
        background-color: #FFFFFF;
        border: 2px solid #FFC1CC;
        border-radius: 6px;
        padding: 4px;
    }
   </string>
  </property>

  <layout class="QVBoxLayout" name="verticalLayout">

   <!-- HEADER -->
   <item>
    <widget class="QLabel" name="label_title">
     <property name="styleSheet">
      <string notr="true">font-size:16pt; font-weight:bold; color:#333333;</string>
     </property>
     <property name="text">
      <string>Edit Video Details</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>

   <!-- Title -->
   <item>
    <widget class="QLineEdit" name="input_title">
     <property name="placeholderText">
      <string>Edit video title</string>
     </property>
    </widget>
   </item>

   <!-- Description -->
   <item>
    <widget class="QTextEdit" name="input_description">
     <property name="placeholderText">
      <string>Edit video description</string>
     </property>
    </widget>
   </item>

   <!-- Genre Dropdown -->
   <item>
    <widget class="QComboBox" name="dropdown_genre">
     <item><property name="text"><string>Travel</string></property></item>
     <item><property name="text"><string>Study</string></property></item>
     <item><property name="text"><string>Comedy</string></property></item>
     <item><property name="text"><string>Fitness</string></property></item>
     <item><property name="text"><string>Lifestyle</string></property></item>
     <item><property name="text"><string>Motivation</string></property></item>
     <item><property name="text"><string>Music</string></property></item>
     <item><property name="text"><string>Technology</string></property></item>
     <item><property name="text"><string>Education</string></property></item>
     <item><property name="text"><string>Entertainment</string></property></item>
     <item><property name="text"><string>Other</string></property></item>
    </widget>
   </item>

   <!-- Buttons -->
   <item>
    <widget class="QPushButton" name="btn_save">
     <property name="text">
      <string>Save Changes</string>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QPushButton" name="btn_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
